core/src/main/java/org/wso2/testgrid/core/command/CleanUpCommand.java

```diff
@@ -131,6 +149,8 @@ public void execute() throws CommandExecutionException {
             }
 
         } catch (IllegalArgumentException e) {
+            throw new CommandExecutionException(e.getMessage(), e);
```

Comment:
You are catching an IllegalArgumentException. Do you know which argument it was? Shouldn't we log it? Exception handling is really important for long-term maintenance of a project.